docs/_includes/code_of_conduct.md

```diff
@@ -0,0 +1,86 @@
+> * Repeated harassment of others. In general, if someone asks you to stop, then stop.
+> * Advocating for, or encouraging, any of the above behaviour.
+
+## Diversity Statement
+
+The MCMCICRO project welcomes and encourages participation by everyone. We are committed to being a community that everyone enjoys being part of. Although we may not always be able to accommodate each individual’s preferences, we try our best to treat everyone kindly.
+
+No matter how you identify yourself or how others perceive you: we welcome you. Though no list can hope to be comprehensive, we explicitly honour diversity in: age, culture, ethnicity, genotype, gender identity or expression, language, national origin, neurotype, phenotype, political beliefs, profession, race, religion, sexual orientation, socioeconomic status, subculture and technical ability, to the extent that these do not conflict with this code of conduct.
+
+Though we welcome people fluent in all languages, MCMICRO development is conducted in English.
+
+Standards for behaviour in the MCMICRO community are detailed in the Code of Conduct above. Participants in our community should uphold these standards in all their interactions and help others to do so as well (see next section).
+
+## Reporting Guidelines
+
+We know that it is painfully common for internet communication to start at or devolve into obvious and flagrant abuse. We also recognize that sometimes people may have a bad day, or be unaware of some of the guidelines in this Code of Conduct. Please keep this in mind when deciding on how to respond to a breach of this Code.
+
+For clearly intentional breaches, report those to the Code of Conduct committee (see below). For possibly unintentional breaches, you may reply to the person and point out this code of conduct (either in public or in private, whatever is most appropriate). If you would prefer not to do that, please feel free to report to the Code of Conduct Committee directly, or ask the Committee for advice, in confidence.
+
+You can report issues to the MCMICRO Code of Conduct committee, at MCMICRO-conduct@googlegroups.com . Currently, the committee consists of:
+
+* [Denis Schapiro](https://github.com/DenisSch) (chair)
+* [Artem Sokolov](https://github.com/ArtemSokolov)
+* [Jeremy Muhlich](https://github.com/jmuhlich)
+
+If your report involves any members of the committee, or if they feel they have a conflict of interest in handling it, then they will recuse themselves from considering your report. Alternatively, if for any reason you feel uncomfortable making a report to the committee, then you can also contact:
+
+* Alyce???
+
+## Incident reporting resolution & Code of Conduct enforcement
+
+*This section summarizes the most important points, more details can be found in [MCMICRO Code of Conduct - How to follow up on a report.](https://github.com/labsyspharm/mcmicro/blob/DenisSch-CODEOFCONDUCT/docs/CODE_OF_CONDUCT_REPORTING.md)*
```

Comment:
The link should probably point to the `master` branch, not `DenisSch-CODEOFCONDUCT`. Otherwise, it will break if/when you delete the branch after merging it.